hooks/FindLocation.tsx

```diff
@@ -1,15 +1,21 @@
 
 const FindLocation = async () => {
+    interface latLng {
+        lat: number,
+        lng: number,
+    }
+    
     interface Location {
         lat: number,
-        lng: number
+        lng: number,
+        city: string
```

Comment:
[pebble] To reduce copy/pasted code (and for future reference), if you ever have an interface that is the exact same as another interface but it just has additional properties, you use that 1st interface instead of copy/pasting its contents by extending it
example:
interface Location extends latLng {
        city: string
        }

Reply:
fixed